app/uk/gov/hmrc/eusubsidycompliancefrontend/views/BecomeAdminConfirmatinPage.scala.html

```diff
@@ -16,37 +16,32 @@
 
```

Comment:
The name of this file should be `BecomeAdminConfirmationPage`

Reply:
Yes, that's done, not sure why this view still remains, will remove.